tests/test_pydev_resolver_plugin.py

```diff
@@ -0,0 +1,254 @@
+import builtins
+from typing import Any
+
+from pytest import fixture, mark, param
+
+from omegaconf import (
+    AnyNode,
+    BooleanNode,
+    Container,
+    DictConfig,
+    EnumNode,
+    FloatNode,
+    IntegerNode,
+    ListConfig,
+    Node,
+    OmegaConf,
+    StringNode,
+    ValueNode,
+)
+from omegaconf._utils import type_str
+from pydevd_plugins.extensions.pydevd_plugin_omegaconf import (
+    OmegaConfDeveloperResolver,
+    OmegaConfUserResolver,
+)
+from tests import Color
+
+
+@fixture
+def resolver() -> Any:
+    yield OmegaConfUserResolver()
+
+
+@mark.parametrize(
+    ("obj", "expected"),
+    [
+        # nodes
+        param(AnyNode(10), {}, id="any:10"),
+        param(StringNode("foo"), {}, id="str:foo"),
+        param(IntegerNode(10), {}, id="int:10"),
+        param(FloatNode(3.14), {}, id="float:3.14"),
+        param(BooleanNode(True), {}, id="bool:True"),
+        param(EnumNode(enum_type=Color, value=Color.RED), {}, id="Color:Color.RED"),
+        # nodes are never returning a dictionary
+        param(AnyNode("${foo}", parent=DictConfig({"foo": 10})), {}, id="any:10"),
+        param(AnyNode("${foo}", parent=DictConfig({"foo": 10})), {}, id="any:10"),
```

Comment:
Duplicated lines + id:
```suggestion
        param(AnyNode("${foo}", parent=DictConfig({"foo": 10})), {}, id="any:inter_10"),
```

Reply:
duplicate id is okay (they get an automatic counter appended). 
Duplicate line, not so much.